nifi-nar-bundles/nifi-zendesk-bundle/nifi-zendesk-processors/pom.xml

```diff
@@ -49,6 +70,16 @@
             <groupId>commons-io</groupId>
```

Comment:
I would remove this dependency since it's only used once in GetZendesk. The IOUtils::toString can be replaced with the following approach.
```
try (InputStream responseBodyStream = response.body();
                     BufferedReader reader = new BufferedReader(new InputStreamReader(responseBodyStream, StandardCharsets.UTF_8))) {
                    final String responseBody = reader.lines().collect(Collectors.joining(System.lineSeparator()));
                    errorMessage = String.format("HTTP %s error for requested URI [%s] with response [%s]", response.statusCode(), uri, responseBody);
                    getLogger().warn(errorMessage);             
}
```